doc/requirements.txt

```diff
@@ -4,3 +4,5 @@ docutils==0.15.2
 packaging==20.9
 jinja2<3.1
 schema==0.7.5
+pyspark==3.3.1
+sagemaker-feature-store-pyspark-3.3
```

Comment:
Is this necessary? These dependencies are specified in the extras. I don't see other ones here https://github.com/aws/sagemaker-python-sdk/tree/master/requirements/extras.